docs/updates/2019-06.md

```diff
@@ -0,0 +1,124 @@
++--------------------------------------+------+--------------------------+----------+-------+
+```
+
+To keep the command shorter, the configuration can be placed in a corectl.yml file. You can read more about the feature
+in the [corectl config](https://github.com/qlik-oss/corectl/blob/master/docs/corectl_config.md#certificates) chapter.
+
+### Unbuild - export app content
+
+![screenshot](../images/corectl-unbuild.gif)
+
+The new experimental command, `unbuild`, exports all dimensions, measures, objects, variables, connections and load
+script from an existing application. As an example, this might be valuable when you want to use existing objects in
+multiple applications or if you would like to extract and version manage application content outside the app itself.
+
+Usage:
+
+```bash
+corectl unbuild -a app-id
+```
+
+This will create a new folder named `app-id-unbuild`. If you would like to use another directory just add `--dir` to
+your command
+
+```bash
+corectl unbuild --dir ./output -a app-id
+```
+
+As always, we are open for feedback. Do you miss anything or do you think we can improve the command further. Don't
+hesitate to reach out to us.
+
+### Variables and bookmarks
+
+We have extended support for application entities to include variables and bookmarks. Variables follow the same
+structure as other advanced commands (`layout`, `ls`, `properties`, `rm`, `set`). Bookmarks is similar but lacks the
+`set` command since selection states can't be moved between different apps.
+
+Example showing properties for a specified variable:
+
+```bash
+corectl variable properties vRunningDate -a "Helpdesk Management.qvf"
+```
+
+```json
+{
+  "qInfo": {
+    "qId": "4b551465-809d-401b-9a82-aabcd892eab4",
+    "qType": "variable"
+  },
+  "qMetaDef": {},
+  "qName": "vRunningDate",
+  "qNumberPresentation": {
+    "qType": "U",
+    "qnDec": 10,
+    "qUseThou": 0
+  },
+  "qDefinition": "=maxstring([Running Date])"
+}
+```
+
+### Import applications
+
+Do you have a couple of Qlik applications you would like to try out in a new Qlik Core container? Do you want to
+automate importing application to an engine instance? The new import command might be helpful for you. The command
+imports a specified application to the engine and returns its provided ID.
+
+Usage:
+
+```bash
+$ corectl app import path/app.qvf
+Imported app with new ID: a0334b7f-e533-4643-9e8c-0e8b6f9fdc84
+```
+
+### Basic completion support for powershell
+
+![screenshot](../images/corectl-powershell.gif)
+
+For those of you favoring Powershell, we have added basic completion support. With the completion enabled you can cycle
+through the available commands and flags for current command.
+
+Please look into the [https://github.com/qlik-oss/corectl#powershell](readme) for more information.
+
+### Other fixes
+
+* Bash completion suggest files based on required file types (e.g. *.json for objects)
```

Comment:
Specification: it is extended to work for commands. It existed for flags before.